doc/developer/build_release.rst.tmpl

```diff
@@ -19,175 +19,165 @@ the first bug fix release, and ``<version>`` can be ``2.0``, ``2.1``, ...
 
 .. _developer_build_release_pre_release_task:
 
-Pre release task
-----------------
-
-Before doing a release, you should merge all the previous branches on this one:
-
-* Merge the release changes (on ``ngeo`` and on ``c2cgeoportal``)
-  to the upper branches i.e.: ``2.2`` => ``2.3``, ``2.3`` => ``2.4`` (master).
-
-  .. note::
-
-     On ``c2cgeoportal`` merge, see if an alembic merge should be done:
+Tasks to do
+-----------
 
-     .. prompt:: bash
+On franch creation (start of the integration phase):
 
-        python3 -m pip install --user -e commons
-        alembic --config=commons/alembic.ini \
-            --name=[main|static] heads
+* Create and initialise the branches
+* Protect the new branches
+* Create the new Transifex ressources
+* Configure dependabot
+* Configure the rebuild
+* Use the ``ngeo`` package linked to the branch
+* Verify that the changelog creation is working
 
-     If yes, create the merge with:
+On release creation:
 
-     .. prompt:: bash
+* Reset the changelog
+* Do the tags
+* Publish it
+* New tests for the upgrade
+* Create the new demo
+* Use the new demo
 
-        alembic --config=commons/alembic.ini \
-            --name=[main|static] \
-            merge --message="Merge <src> and <dst> branches" \
-            <rev 1> <rev 2>
+.. note::
 
-     Remove the import and replace the core of the method by ``pass`` in the generated file.
+   I don't specify but all your changes sould be commited
```

Comment:
I am not sure to understand. Do we want to say "Do not forget to commit all your changes." ?

Reply:
"I don't specify but all changes should be committed."

Is it clearer?

Reply:
No I don't think so... what is it that do you not specify?

Reply:
Every time you have to commit changes.

Reply:
So why don't we say "All changes should be committed" or "Do not forget to commit all your changes"

"I don't specify" sounds really strange in a doc.